AK/StringUtils.cpp

```diff
@@ -462,6 +462,20 @@ String to_titlecase(StringView str)
     return builder.to_string();
 }
 
+String invert_case(StringView str)
+{
+    StringBuilder builder;
```

Comment:
Use this, since you know the length the string is going to be.
```suggestion
    StringBuilder builder(str.length());
```

Reply:
Is there really a benefit to this? Most of the other StringBuilder's in the code base, including in the functions directly above this one, just use StringBuilder builder. If there is benefit I'm all ears but not sure it makes a difference.

Reply:
Theoretically, the larger the string becomes, the more allocations it saves (1 allocation as opposed to however many times it would have to grow the buffer, looks like it's log2(n) from StringBuilder::will_append). In practice probably not really, especially when used in the vim emulation, but imo it's a free optimization that, if anything, makes the code clearer (implies the string builder will be the same size as the passed in string).

Reply:
Ya, figured it had to do with multiple allocation as the buffer grew just wasn't convinced it was really worth anything. Easy change, will do, thanks!